src/components/[guild]/Requirements/components/RequirementImage.tsx

```diff
@@ -0,0 +1,52 @@
+import { Circle, Img, SkeletonCircle, useColorMode } from "@chakra-ui/react"
+
+type Props = {
+  isImageLoading?: boolean
+  image?: string | JSX.Element
+  withImgBg?: boolean
+}
+
+const RequirementImage = ({ isImageLoading, withImgBg, image }: Props) => {
```

Comment:
I think if we remove the `OriginalRequirementPreview` component as I mentioned in my other comment, we won't need this one, right? We could just conditionally render the image editor inside `Requirement.tsx`. What do you think?

Reply:
Nevermind, now I see we can use this component in 3 different places!